src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerGenotypingEngine.java

```diff
@@ -167,6 +168,7 @@ public CalledHaplotypes assignGenotypeLikelihoods(final List<Haplotype> haplotyp
                             loc);
 
             VariantContext mergedVC = AssemblyBasedCallerUtils.makeMergedVariantContext(eventsAtThisLocWithSpanDelsReplaced);
+            int mergedAlleles = mergedVC.getAlleles().size();
```

Comment:
This variable name isn't helpful.  Maybe something like "numberOfAllelesBeforeSubsetting" would be clearer